userspace/libscap/syscall_info_table.c

```diff
@@ -29,14 +29,19 @@ static struct ppm_syscall_desc g_syscall_info_table[PPM_SC_MAX];
 
 static void load_syscall_info_table() {
 	const char *sc_names[PPM_SC_MAX] = {
-#define PPM_SC_X(name, value) #name,
+#define PPM_SC_X(name, value) [value] = #name,
```

Comment:
Note: i agree with #816 about renaming syscall info table (that should really be `ppm_sc_names_table`) and drop category.

I'd leave this job to that PR.